### ISSUE TITLE
Collaboration with ESLint

### ISSUE THREAD

**Issue by IanVS** [willklein/dryer#15]
ESLint is talking about doing something very similar to this project (see https://github.com/eslint/eslint/issues/2203#issuecomment-120438720), and I'm trying to think of ways it could be done.  It was suggested to look at what you're doing here and possibly combine efforts.  Are you interested in working together in some manner?

**Comment by willklein** [comment]
@IanVS, absolutely! I was just tripped over that thread too, thank you for referencing it. I think I have a clear idea on how to do everything, but I would be so happy to get your ideas and evolve the design into something even better. I would also love to be working with you and others on the implementation too. :smile: 

I originally built this to make it easier to get off the ground with an existing code base. I also have ideas to keep it relevant for maintaining configs, so it's not just a bootstrapping tool. With that in mind, I decided to pick this project back up yesterday, started refactoring the code last night before writing some tests.

I also just noticed `eslint-nibble`. Great idea! This shares a lot of what I'm interested in: building tools (nibble, dryer) to make tools (ESLint) easier to use.

Let me know if we can chat this weekend, or next week, depending on when you like to do things.

**Comment by willklein** [comment]
After reading up on [eslint #2203](https://github.com/eslint/eslint/issues/2203#issuecomment-120438720), it sounds like this would move under eslint, as something like "eslint-autoconfig." I'm fine with that.
